app/frontend/styles/globals.css

```diff
@@ -3,12 +3,19 @@
 @tailwind components;
 @tailwind utilities;
 
+
+html footer {@apply bg-gray-500}
+
+html body{ @apply bg-gray-200}
+
+div.timebook {@apply my-10 rounded-xl border-2 border }
+
```

Comment:
Our plan was to keep this file as small as possible.
Can we move this code to where it is used?